WordPress/Classes/System/WordPressAppDelegate+openURL.swift

```diff
@@ -20,6 +20,22 @@ import AutomatticTracks
             return true
         }
 
+        /// WordPress only. Handle deeplink from JP that requests data export.
```

Comment:
Would it be helpful to wrap these in a `AppConfiguration.isWordPress` call? I can add it if you think that's a good idea.

Reply:
Good idea. It's guarded in the route's `perform` method, but if it bounces there would be a potential logic error there because the next line after `handle` returns true.

This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/19710/commits/af22570931c20bf9951f44ec5f0d1cb9716d21dc